pkg/backup/backup_pv_action.go

```diff
@@ -32,19 +33,19 @@ type backupPVAction struct {
 	log logrus.FieldLogger
 }
 
-func NewBackupPVAction(logger logrus.FieldLogger) ItemAction {
+func NewBackupPVAction(logger logrus.FieldLogger) interfaces.BackupItemAction {
```

Comment:
something I often wonder/go back and forth about is -- in a scenario like this - should we return an interface type, or should we export `type backupPVAction struct` and return it?  There is the old adage of `accept interfaces, return concrete types`.  Not sure it matters much here, but curious what others think about this.

Reply:
I have a dream: to clean up our entire code to always return a type and never an interface. 

I would like to make backupPVAction external then.

Reply:
In something that's public facing, I think returning a concrete type is a good idea if we can.

Reply:
I'll comb thru this code and make this change.